docs/topics/logging.txt

```diff
@@ -117,27 +117,71 @@ of a Python formatting string containing
 :ref:`LogRecord attributes <python:logrecord-attributes>`; however,
 you can also write custom formatters to implement specific formatting behavior.
 
+.. _logging-how-to:
+
 How to use logging
 ==================
 
-Once you have configured your loggers, handlers, filters and
-formatters, you need to place logging calls into your code. Using the
-logging framework works like this::
+Django provides a :ref:`default logging configuration
+<default-logging-configuration>`, that for example generates the messages that
+appear in the console when using the :djadmin:`runserver`.
+
+Make a basic logging call
+-------------------------
+
+Python programmers will often drop ``print()``\s into their code as a quick
+and easy debugging tool. Using the logging framework is only a little more
+effort than that, but it's much more elegant and flexible. As well as being
+useful for debugging, you can also use logging to provide you with lot more
+information about the state and health of your application.
```

Comment:
We trying to to avoid words that minimize the difficulty involved in a task or operation, such as "easy" (see [Writing style](https://docs.djangoproject.com/en/3.2/internals/contributing/writing-documentation/#writing-style)). Maybe we can chop them here :thinking: 

```suggestion
Python programmers will often drop ``print()``\s into their code as a debugging
tool. Using the logging framework is only a little more effort than that, but
it's much more elegant and flexible. As well as being useful for debugging, you
can also use logging to provide you with lot more information about the state
and health of your application.
```